content-injection-from-rule/src/App.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import algoliasearch from 'algoliasearch/lite';
+import {
+  InstantSearch,
+  SearchBox,
+  Configure,
+  RefinementList,
+  Pagination,
+  Highlight,
+  Snippet,
+} from 'react-instantsearch-dom';
+import PropTypes from 'prop-types';
+
+import { InjectedHits } from './InjectedHits';
+import { FreeReturnPolicy } from './FreeReturnPolicy';
+
+import './App.css';
+
+const searchClient = algoliasearch(
+  '5WNRH2J3IK',
+  '64e897c7e06d86a32c3a8d409b403756'
+);
```

Comment:
Currently the Rule lives in this Algolia app. What is the recommended method here? Should I add the Rule in `latency` instead?